packages/features/ee/managed-event-types/lib/handleChildrenEventTypes.ts

```diff
@@ -244,22 +242,39 @@ export default async function handleChildrenEventTypes({
       .reduce((newObj, [key, value]) => ({ ...newObj, [key]: value }), {});
     console.log({ unlockedFieldProps });
     // Update event types for old users
-    const oldEventTypes = await prisma.$transaction(
-      oldUserIds.map((userId) => {
-        return prisma.eventType.update({
-          where: {
-            userId_parentId: {
+    const oldEventTypes = await prisma.$transaction(async (prisma) => {
+      return await Promise.all(
+        oldUserIds.map(async (userId) => {
+          const existingEventType = await prisma.eventType.findFirst({
+            where: {
               userId,
               parentId,
             },
-          },
-          data: {
-            ...updatePayloadFiltered,
-            hashedLink: "hashedLink" in unlockedFieldProps ? undefined : hashedLinkQuery(userId),
-          },
-        });
-      })
-    );
+          });
+
+          const existingHashedLink = await prisma.hashedLink.findFirst({
```

Comment:
My best guess is you added these extra queries and haven't mocked their values @Souptik2001

Reply:
@Amit91848 But when I do a console log inside this `prisma.$trasaction`, that console log is not printed only, while if I do it inside other `prisma.$transaction` they are printed.

One thing - if you see I have changed how this `prisma.$transaction` is used. I.e passed a function instead of array of `PrismaPromises` like other usecases. So, do you think this may be because of that?

Reply:
@Amit91848 @anikdhabal any help on this? 😁

Reply:
> One thing - if you see I have changed how this `prisma.$transaction` is used. I.e passed a function instead of array of `PrismaPromises` like other usecases. So, do you think this may be because of that?

Worth a try, hadn't noticed this before. AFAIK transaction expects an array of queries to be executed sequentially